Framework/Core/src/CommonServices.cxx

```diff
@@ -517,32 +517,30 @@ auto sendRelayerMetrics(ServiceRegistry& registry, DataProcessingStats& stats) -
   stats.lastReportedPerformedComputations.store(stats.performedComputations.load());
   O2_SIGNPOST_END(MonitoringStatus::ID, MonitoringStatus::SEND, 0, 0, O2_SIGNPOST_BLUE);
 
-  // Things which we report every 30s
-  if (timeSinceLastLongUpdate < 30000) {
-    return;
-  }
-
   auto device = registry.get<RawDeviceService>().device();
 
-  stats.channelBytesIn.resize(device->fChannels.size());
-  stats.channelBytesOut.resize(device->fChannels.size());
-  size_t ci = 0;
+  auto lastTotalBytesIn = 0;
+  auto lastTotalBytesOut = 0;
+  stats.totalBytesIn.exchange(lastTotalBytesIn);
+  stats.totalBytesOut.exchange(lastTotalBytesOut);
+  auto totalBytesIn = 0;
+  auto totalBytesOut = 0;
```

Comment:
The auto variables here are not safe I think. They will become integers, but we want uint64_t.